scribus/plugins/tools/hunspellcheck/hunspellpluginimpl.cpp

```diff
@@ -181,7 +181,7 @@ bool HunspellPluginImpl::parseTextFrame(StoryText *iText)
 			wf.replacements = hspellerMap[wordLang]->suggest(word);
 			wordsToCorrect.append(wf);
 		}
-		currPos = iText->nextWord(wordStart);
+		currPos = iText->endOfWord(wordStart);
```

Comment:
Yes, I was under the assumption that we are not touching that unused code at all, but if we need to modify it because it is broken then we are better of just killing it.